src/coreComponents/events/EventManager.cpp

```diff
@@ -207,6 +208,7 @@ bool EventManager::run( DomainPartition & domain )
       }
     }
 
+    section.endSection();
```

Comment:
@arng40 Section end output must be optionally able to include information, in order to support PR https://github.com/GEOS-DEV/GEOS/pull/3303 modifications